src/apps/service/apps/combined/jobs.clj

```diff
@@ -74,9 +74,10 @@
        (validate-job-steps app-id)
        (map (partial build-job-step-save-info job-id))))
 
-(defn- prepare-de-job-step-submission
-  [job-info job-step submission]
+(defn- prepare-job-step-submission-common
+  [job-info job-step {:keys [config] :as submission}]
   (assoc submission
+    :config               (:job_config submission config)
```

Comment:
Here is where the new `job_config` key replaces the user's original `config` for combined-pipeline job submission JSON.